srcpkgs/wishlist/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'wishlist'
+pkgname=wishlist
+version=0.15.1
+revision=1
+build_style=go
+go_import_path="github.com/charmbracelet/wishlist"
+hostmakedepends="pkg-config"
+short_desc="SSH directory"
+maintainer="zenobit <zenobit@disroot.org>"
+license="MIT"
+homepage="https://github.com/charmbracelet/wishlist"
+#changelog=""
+distfiles="https://github.com/charmbracelet/wishlist/archive/refs/tags/v${version}.tar.gz"
+checksum=a6d1e1dd526cc4cf5b5f5152683a0731636d189da005c36b68dffe24910e2f31
+
+do_build() {
+	go build ./cmd/wishlist/
```

Comment:
```
go_package="${go_import_path}/cmd/wishlist"
```
instead of overwriting `do_build` is usually how its done and probably works.

Reply:
Not this time

Usually I am trying to stick with build as provided from upstream if that work on Void...